tests/example-all-fail/expected_results.json

```diff
@@ -6,7 +6,8 @@
       "name": "Bob #hey responds to stating something",
       "test_code": "Bob.hey(\"Tom-ay-to, tom-aaaah-to.\").should eq \"Whatever.\"",
       "status": "fail",
-      "message": "Expected: \"Whatever.\"      got: \"Whatevah.\""
+      "message": "Expected: \"Whatever.\"      got: \"Whatevah.\"",
+      "task_id": 1
```

Comment:
There are no tasks defined for this exercise, so this should be omitted.

Reply:
Isnt it just ensuring the result if the solution is incorrect?

Reply:
Hmmm, I missed that nesting is already used: https://github.com/exercism/crystal-test-runner/blob/main/tests/example-all-fail/spec/mini_bob_spec.cr

Have to think about this for a bit. I'll need to do some testing.

Reply:
Hmmm, I missed that nesting is already used: https://github.com/exercism/crystal-test-runner/blob/main/tests/example-all-fail/spec/mini_bob_spec.cr

Have to think about this for a bit. I'll need to do some testing.

Reply:
Only 1 exercise uses that nesting atm and it is likely that exercise will shift away from it

Reply:
Ah. For non-nested tests, does it output as `task_id` of `nil` (or omit the key)?

Reply:
It omits the key

Reply:
Brilliant